relay-cardinality/src/limiter.rs

```diff
@@ -20,19 +19,27 @@ pub struct Scoping {
     pub project_id: ProjectId,
 }
 
+/// Accumulator of all cardinality outcomes.
+pub trait Outcomes {
```

Comment:
Do you actually need a trait on this level? Both implementations wrap a HashSet, so could we just make the set element a type parameter?

If we keep the trait: "outcome" is almost like a reserved word at sentry, can we call it `Rejections`?

Reply:
Yeah, it could just be a `HashSet`, the idea was that in the future we can do `outcomes.reject(entry, quota)` when we need the quota.

Also with the inner value of EntryId being hidden and we need the inner value in the HashSet I came up with this trait.